### ISSUE TITLE
Notification when a when grade is published.

### ISSUE THREAD

**Issue by apomalyn** [ApplETS/Notre-Dame#29]
**Is your feature request related to a problem? Please describe.**
As a user, I would like to receive a notification when a new grade is published on SignETS

**Additional context**
The ETS use AWS SNS to push events. In v3 we used this [lib](https://github.com/ApplETS/ETSMobile-Notifications-Android)

**Comment by AlexandreMarkus** [comment]
https://github.com/aws-amplify/amplify-flutter/issues/319

**Comment by AlexandreMarkus** [comment]
https://firebase.flutter.dev/docs/messaging/server-integration/

**Comment by zaclimon** [comment]
Just leaving that out there since it will prove itself a little 🌶️ to work with:
https://github.com/ApplETS/Notre-Dame-v3/wiki/Push-Notifications-(EN)

**Comment by AlexandreMarkus** [comment]
https://pub.dev/packages/aws_sns_api

**Comment by apomalyn** [comment]
Alright, after some work with @AlexandreMarkus we are blocked. Here is a resume of what we've learned.
- You can find the documentation on how it was done in v2, v3 beta for Android [here](https://github.com/ApplETS/ETSMobile-Notifications-Android/tree/750e2a4ea505cd476f953d97e4761f34f72b758a) and to read [this article](https://github.com/ApplETS/Notre-Dame-v3/wiki/Push-Notifications-(EN)) written by @Sonphil by the time (thanks again to him)
- We will need to connect Firebase AND AWS SNS at the same time. AWS SNS to receive a notification.
- Currently, AWS doesn't support SNS for Flutter. Maybe we could implement this natively directly or do a plugin

**Comment by MysticFragilist** [comment]
![image](https://user-images.githubusercontent.com/25663435/142032984-e5283cac-150a-4307-8c31-f05f6fc0fc13.png)
:eyes:

**Comment by MysticFragilist** [comment]
![image](https://user-images.githubusercontent.com/25663435/156952331-f8e126cb-f786-450d-afba-6be68edfab34.png)
Grrr 😠 it's an emotional rollercoaster at this point haha

**Comment by MysticFragilist** [comment]
Somebody made a tutorial for handling the android part! https://medium.com/iiitians-network/flutter-push-notifications-using-aws-sns-dac464c1edf0 we will be able to start this issue on android! 😄

**Comment by MysticFragilist** [comment]
yep! I won't personally do it now, but if any new member want to start on that, at least it's not blocked anymore ;)

**Comment by MysticFragilist** [comment]
Isn't blocked anymore, community built out a new aws sns flutter package https://pub.dev/packages/aws_sns_api. I'm going to work on notification form now on.

**Comment by XavierPaquet-Rapold** [comment]
Blocked. After investigation, the notification service used by signÉTS doesn't work anymore 😢

**Comment by XavierPaquet-Rapold** [comment]
prise, puisque les notifications ne sont plus une priorité pour le club et que le SNS donnait des erreurs de leur côté. Si cette tâche revient dans nos priorités, il va falloir contacter les services TI pour l'envoi de notification.
